pkg/minikube/cluster/cluster.go

```diff
@@ -368,9 +368,6 @@ func getDriverOptions(config MachineConfig) interface{} {
 	switch config.VMDriver {
 	case "virtualbox":
 		driver = createVirtualboxHost(config)
-	case "vmwarefusion":
-		fmt.Println("VMWare Fusion driver will be deprecated soon. Please consider using other drivers.")
-		driver = createVMwareFusionHost(config)
```

Comment:
That means we are not even supporting vmwarefusion :(, Recently https://github.com/machine-drivers/docker-machine-driver-vmware is already added and can be used.

Reply:
@praveenkumar Ah. I may have misunderstood the issue then. So we were just looking to remove the deprecation warnings themselves and not the different cases / functions that implemented vmwarefusion?

If that's the case I can roll back those changes. But has the updated driver been implemented already?